easybuild/easyconfigs/g/GDAL/GDAL-2.0.0-intel-2015a.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.0.0'
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+    ('netCDF', '4.3.3.1'),
+    ('expat', '2.1.0'),
+    ('libxml2', '2.9.2', '-Python-2.7.9'),
```

Comment:
does GDAL require the Python-enabled version of libxml2?

if so, use a versionsuffix for GDAL and use Python 2.7.10 (easy fix)

if not, use the non-Python version?